tests/util/test_result.rs

```diff
@@ -0,0 +1,82 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+extern crate rustc_serialize;
+extern crate libstratis;
+
+use libstratis::engine::EngineError;
+use self::rustc_serialize::json::ParserError;
+use std::fmt;
+use std::io::Error;
+
+#[derive(Debug)]
+pub enum TestErrorEnum {
+    _Ok,
```

Comment:
Might as well get rid of constructors with underscores. Also, it is possible that ```_Ok``` will always be unnecessary, for Ok is by definition not an error.